rtwo/drivers/openstack_network.py

```diff
@@ -242,13 +263,24 @@ def rename_security_group(self, project, security_group_name=None):
     ##Libcloud-Neutron Interface##
     @classmethod
     def lc_driver_init(self, lc_driver, *args, **kwargs):
-        lc_driver_args = {
-            'username': lc_driver.key,
-            'password': lc_driver.secret,
-            'tenant_name': lc_driver._ex_tenant_name,
-            #Libcloud requires /v2.0/tokens -- OS clients do not.
-            'auth_url': lc_driver._ex_force_auth_url.replace('/tokens',''),
-            'region_name': lc_driver._ex_force_service_region}
+        if kwargs.has_key('ex_force_auth_token'):
```

Comment:
Lets try to avoid `if token, X; else Y` statements because it will make the codebase harder to reason about.

Instead, add `token` and `endpoint_url` via `token = getattr(lc_driver, '_ex_force_auth_token')` and make a single `lc_driver_args` with all the common key-value pairs.